<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Alekh" failures="1" tests="1" name="Test_Runner.PublicDonationRunner" time="96.124" errors="0" timestamp="2024-03-01T13:30:59 IST" skipped="0">
  <testcase classname="Test_Runner.PublicDonationRunner" name="runScenario" time="96.124">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at Stepdefination.PublicDonation.the_use_should_re_direct_to_public_donation_page(PublicDonation.java:62)
at âœ½.the use should re-direct to public donation page(file:///C:/Users/Dell/git/repository/com.affnetz/src/test/java/Feature_files/publicDonation.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- Test_Runner.PublicDonationRunner -->
